sections/404-error.liquid

```diff
@@ -0,0 +1,61 @@
+<header class="section-header
+    parallax-element
+    section-header--image" style="background-image: url({{ section.settings.background-404 | img_url: 'master' }})">
+    <div class="text-wrapper--center-cell text-max-width">
+        <div class="text-wrapper--small text-center">
+            <h1 class="section-header__title text-center">{{ 'general.404.title' | t }}</h1>
+        </div>
+    </div>
+</header>
```

Comment:
Can't this just reuse the collection styling classes? text-wrapper--center-cell, text-max-width? we shouldn't need the inline styles or the new style classes if we do that. 

https://github.com/Blendtec/blendtec-shopify-theme/blob/b124c51801052de660d80bc00c1e045e6e4a029a/sections/collection-template.liquid#L27